runtime/memcache.h

```diff
@@ -25,9 +21,9 @@ mixed mc_get_value(const char *result_str, int32_t result_str_len, int64_t flags
 
 bool mc_is_immediate_query(const string &key);
 
-
-constexpr int64_t MEMCACHE_SERIALIZED = 1;
-constexpr int64_t MEMCACHE_COMPRESSED = 2;
+constexpr inline auto MEMCACHE_SERIALIZED = static_cast<int64_t>(1U << 0U);
```

Comment:
what's wrong with `inline constexpr int64_t MEMCACHE_SERIALIZED = 1U << 0U`?

Reply:
Nothing wrong :) Can you add more details to your question?

Reply:
`auto` for `auto` and also `static_cast`.
It's not clear why it's there. I suggest not to use it here, but to make it explicit. In general, like other `inline constexpr` in the code